docs/tempo/website/getting-started/_index.md

```diff
@@ -6,18 +6,31 @@ weight: 100
 # Getting started with Tempo
 
 Distributed tracing visualizes the lifecycle of a request as it passes through
-an application. There are a few components that must be configured in order to get a
+a set of applications. There are a few components that must be configured in order to get a
 working distributed tracing visualization.
 
+An example trace visualized in Grafana:
+
+<p align="center"><img src="../example-trace.png" alt="Example Trace"></p>
+
+This document discusses the four major pieces necessary to build out a tracing system: 
+Instrumentation, Tracing Pipeline, Tracing Backend and Visualization. If one were to build a diagram laying 
```

Comment:
Thoughts on removing the Tracing word from the Pipeline and the Backend? Seems a bit reduntant.